spec/models/offspring_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.describe Offspring, type: :model do
+  active_record_offspring = 'activerecord.errors.models.offspring.attributes'
+
+  after(:each) do
+    Warden.test_reset!
+  end
+  describe "validating attribute first_name " do
+    it "is invalid without first_name" do
+      off = FactoryGirl.build(:offspring, first_name: "")
+      off.valid?
+      expect(off.errors[:first_name]).to include(I18n.t('f_name.blank', scope: active_record_offspring))
```

Comment:
Active Record has a i18n structure that you need to respect. f_name is not anything at all, and this will mean that if you change language you won't get translations